src/contexts/WalletContext/services.ts

```diff
@@ -18,14 +18,18 @@ const unavailableText = 'Service unavailable, please try again shortly';
  * @param provider - The ethers-wrapped provider
  */
 export const checkForCorrectNetwork = async (provider: ethers.providers.JsonRpcProvider) => {
-  const network = await provider.getNetwork();
-  if (network.name !== process.env.REACT_APP_REQUIRED_ETHEREUM_NETWORK) {
-    // eslint-disable-next-line
-    console.warn(
-      `User wallet is connected to '${network.name}' instead of '${
-        process.env.REACT_APP_REQUIRED_ETHEREUM_NETWORK || '<unknown>'
-      }'`,
-    );
+  try {
+    const network = await provider.getNetwork();
+    if (network.name !== process.env.REACT_APP_REQUIRED_ETHEREUM_NETWORK) {
+      // eslint-disable-next-line
+      console.warn(
+        `User wallet is connected to '${network.name}' instead of '${
+          process.env.REACT_APP_REQUIRED_ETHEREUM_NETWORK || '<unknown>'
+        }'`,
+      );
+      throw new Error();
```

Comment:
```suggestion
      throw new Error(`User wallet is connected to '${network.name}' instead of '${
          process.env.REACT_APP_REQUIRED_ETHEREUM_NETWORK || '<unknown>'
        }'`);
```

remove not needed console.warn and eslint-disable-next-line... Enrich the Error you throw

Reply:
done in ff692256e500f567310f338bb4a684c73a01433a